cves/kernel/CVE-2017-7487.yml

```diff
@@ -230,13 +229,13 @@ interesting_commits:
 
     Example interesting commits:
       * Mentioned as a problematic commit in the past
-        e.g. "This fixes regression in commit xys"
+        e.g. 'This fixes regression in commit xys'
       * A significant rewrite in the git history
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
   - commit:
-    note:
+    note: 
```

Comment:
no interesting commits?

Reply:
The only two commits which address this code are the original creation of the vulnerability and the fix.